### PR TITLE
test: test-cases: RHOAM 1.2.0

### PR DESCRIPTION
**Pull request by psturc**
# Description
Update of test cases for RHOAM 1.2.0

### CODE REVIEW

File: test-cases/tests/alerts/c19-validate-creation-of-invalid-username-triggers-alert.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+7. Create username `alongusernamethatisabovefourtycharacterslong2` and hit Save
+8. Go to `Credentials` tab, fill in password and switch `Temporary` to OFF, hit Set password
+9. In Anonymous window, go to OpenShift console and login via testing-idp as `alongusernamethatisabovefourtycharacterslong2` user
+10. Go to alert manager and log in as kubeadmin
+
+```bash
+open "https://$(oc get route alertmanager-route -n redhat-rhoam-middleware-monitoring-operator -o jsonpath='{.spec.host}')"
+```
+
+> Validate there is "ThreeScaleUserCreationFailed" alert firing
```
Comment (laurafitzgerald) [inline test-cases/tests/alerts/c19-validate-creation-of-invalid-username-triggers-alert.md:55]: @psturc the logic to resolve the alert when the user is deleted is now in place. could we add the following steps.

- `oc delete user <username>`
- verify the alert has resolved for that username.

Comment (laurafitzgerald) [reply test-cases/tests/alerts/c19-validate-creation-of-invalid-username-triggers-alert.md:55]: i wonder if we should also test the code changes added by @philbrookes by deleting the idp when this is done and seeing that the users are deleted from 3scale on the next reconcile?

Comment (philbrookes) [reply test-cases/tests/alerts/c19-validate-creation-of-invalid-username-triggers-alert.md:55]: This seems like a good idea.

File: test-cases/tests/installation/a30-validate-installation-of-rhoam-addon-and-integration-with-ld.md [hunk 2]
```diff
@@ -84,8 +84,6 @@ watch "oc get rhmi rhoam -n redhat-rhoam-operator -o json | jq .status.stage"
 ```
 
 6. Once the status is "completed", the installation is finished and you can go to another step
-7. Go to `Monitoring` section of your cluster in OCM UI
-   > Make sure no alerts are firing
```
Comment (psturc) [inline test-cases/tests/installation/a30-validate-installation-of-rhoam-addon-and-integration-with-ld.md:88]: Monitoring tab is no longer present in OCM UI

**Comment by trepel** [comment]
I went through it and I think it is OK

/lgtm

**Pull request merged**